ambry-store/src/test/java/com.github.ambry.store/BlobStoreTest.java

```diff
@@ -279,6 +286,80 @@ public void cleanup() throws InterruptedException, IOException, StoreException {
     assertTrue(tempDir.getAbsolutePath() + " could not be deleted", StoreTestUtils.cleanDirectory(tempDir, true));
   }
 
+  private void shutdownStoreAndDeleteFiles() throws IOException, StoreException {
+    if (store.isStarted()) {
+      store.shutdown();
+    }
+    assertTrue(tempDir.getAbsolutePath() + " could not be deleted", StoreTestUtils.cleanDirectory(tempDir, true));
+  }
+
+  private BlobStore createBlobStore(ReplicaId replicaId) {
+    StoreConfig config = new StoreConfig(new VerifiableProperties(properties));
+    return createBlobStore(replicaId, config, null);
+  }
+
+  private BlobStore createBlobStore(ReplicaId replicaId, StoreConfig config,
```

Comment:
i'd prefer to have all these helper functions after all the real tests - makes it easier to read IMO (instead of having helpers mixed up with @Test decorated functions)

Reply:
I just threw them where the previous helper methods were.  Want them all below the tests?  I agree that it would look nicer too.

Reply:
Yeah all below tests will be nice.

Reply:
Moved